### PR TITLE
readme updated: daemon run instructions added, obsolete info removed

### PR DESCRIPTION
**Pull request by milyin**
- added instruction how to run daemon with plugin
- removed github npm repositiory instruction

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,104 +1,166 @@
...
-          }
-        }
-        // Optionally, add the REST plugin
-        rest: { http_port: 8000 }
-      }
-    }
-    ```
-  - Run the zenoh router with:
-    `zenohd -c zenoh.json5`
+   Expected output is like:
```
Comment (Charles-Schleich) [inline README.md:99]:
```suggestion
   Expected output should be something similar to
```

File: README.md [hunk 2]
```diff
@@ -1,104 +1,166 @@
...
 
-Or added via package.json
+- [NPM](https://www.npmjs.com/package/npm)
+- [yarn](https://classic.yarnpkg.com/lang/en/docs/install/#debian-stable)
+- [Typescript](https://www.typescriptlang.org/download/) 
 
-`"@eclipse-zenoh/zenoh-ts": "0.0.8" `
+2. Navigate to the directory `zenoh-ts`
+
+1. Run the commands:
```
Comment (Charles-Schleich) [inline README.md:121]:
```suggestion
1. Navigate to the directory `zenoh-ts`

2. Run the commands:
```

File: README.md [hunk 3]
```diff
@@ -1,104 +1,166 @@
...
 
+This library is currently compatible with browsers, but not with NodeJS due to websocket library limitations.
+To run the command line examples use the javascript runtime [deno](https://deno.com/) which is expected to be consistent with the browser.
 
-[zenoh]: https://github.com/eclipse-zenoh/zenoh
+1. Install [deno](https://deno.com/)
+1. Navigate to the `zenoh-ts/examples/deno` directory
+1. Install the `zenoh-ts` library by running `yarn install`
+1. Run zenohd with the remote_api plugin, configured to websocket port 10000, as described above
+1. Run the examples by running `yarn example <PATH TO EXAMPLE>`, i.e. `yarn example src/z_sub.ts`
```
Comment (Charles-Schleich) [inline README.md:137]:
```suggestion
1. Install [deno](https://deno.com/)
2. Navigate to the `zenoh-ts/examples/deno` directory
3. Install the `zenoh-ts` library by running `yarn install`
4. Run zenohd with the remote_api plugin, configured to websocket port 10000, as described above
5. Run the examples by running `yarn example <PATH TO EXAMPLE>`, i.e. `yarn example src/z_sub.ts`
```

File: README.md [hunk 4]
```diff
@@ -1,104 +1,166 @@
...
 
+To run publisher and subscriber examples:
+
+```sh
+yarn example src/pub.rs
+```
 
+```sh
+yarn example src/sub.rs
```
Comment (Charles-Schleich) [inline README.md:146]:
```suggestion
yarn example src/z_pub.ts
```

```sh
yarn example src/z_sub.ts
```

File: EXAMPLE_CONFIG.json5
Comment (Charles-Schleich) [inline EXAMPLE_CONFIG.json5:1]: Should we not also mention that the user can add configuration for certificates for secure websockets here ? 
```
  secure_websocket: {
        certificate_path: 'path/to/certificate_path',
        private_key_path: 'path/to/private_key_path'
      }
```

**Review by Charles-Schleich — COMMENTED** [review]
Small changes after proof reading

File: EXAMPLE_CONFIG.json5
Comment (milyin) [reply EXAMPLE_CONFIG.json5:1]: The goal was to allow to user to run the examples immediately, without making any changes. The example with certificates is in "config.json5" file, I'll add reference to it to readme.
There is no way to comment out part of json config :-(

File: README.md (hunk 1 above)
Comment (milyin) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (milyin) [reply README.md]: fixed - it all shoud be 1. 1. 1. - md does numbering automatically

File: README.md (hunk 3 above)
Comment (milyin) [reply README.md]: same thing. Though yes, I agree that if user read this with e.g. vim, better to make correct numberring. Fixed

File: README.md (hunk 4 above)
Comment (milyin) [reply README.md]: fixed

**Review by Charles-Schleich — APPROVED** [review]

**Pull request merged**